recipes/rb-net-http-persistent/LICENSE.txt

```diff
@@ -0,0 +1,9 @@
+(The MIT License)
```

Comment:
@svengiegerich, please remove the license. :)

Reply:
oh, you're right. strange. could you open a PR in the upstream repo and move the license into a separate file.

(not blocking for this PR, but would be nice)